This fixes build issues on stricter environments (e.g. with musl libc),
where `nullptr_t` is not available in the global namespace.
--- a/Telegram/lib_ui/ui/widgets/buttons.cpp
+++ b/Telegram/lib_ui/ui/widgets/buttons.cpp
@@ -803,7 +803,7 @@ SettingsButton::SettingsButton(
 
 SettingsButton::SettingsButton(
 	QWidget *parent,
-	nullptr_t,
+	std::nullptr_t,
 	const style::SettingsButton &st)
 : RippleButton(parent, st.ripple)
 , _st(st)
--- a/Telegram/lib_ui/ui/widgets/buttons.h
+++ b/Telegram/lib_ui/ui/widgets/buttons.h
@@ -12,6 +12,7 @@
 #include "ui/text/text.h"
 #include "styles/style_widgets.h"
 
+#include <cstddef>
 #include <memory>
 
 class Painter;
@@ -276,7 +277,7 @@ public:
 		const Text::MarkedContext &context = {});
 	SettingsButton(
 		QWidget *parent,
-		nullptr_t,
+		std::nullptr_t,
 		const style::SettingsButton &st = st::defaultSettingsButton);
 	~SettingsButton();
 
-- 
2.50.1

